### DISCUSSION TITLE
eworldtrade.com

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#1389] [Personal Filter]
https://www.eworldtrade.com/c/ew73142347/

popup ini selalu muncul, annoyance
![image](https://github.com/realodix/AdBlockID/assets/19517680/7f8a32a7-721e-419e-b4e9-ffdeb9ceac6c)

minta tolong bikinkan filternya,

**Comment by 3xploiton3** [comment]
![image](https://github.com/realodix/AdBlockID/assets/19517680/34d01152-9c50-4c0a-b28e-59a319ccad0d)

**Reply by realodix** [reply]
Udah sekitar 10 menit muter2, Saya gak berhasil menemukannya untuk popup yang ini

**Comment by realodix** [comment]
> https://www.eworldtrade.com/c/ew73142347/
> 
> popup ini selalu muncul, annoyance ![image](https://user-images.githubusercontent.com/19517680/238135609-7f8a32a7-721e-419e-b4e9-ffdeb9ceac6c.png)
> 
> minta tolong bikinkan filternya,


ini bisa `eworldtrade.com###modalExitIntent` atau `eworldtrade.com##.exit-intent-popup-ssn`

atau kalau digabungin bisa `eworldtrade.com###modalExitIntent.exit-intent-popup-ssn`

Tergantung kebutuhan aja nanti, karena dilihat dulu, takutnya kenal popup lainnya yang mungkin penting

**Reply by realodix** [reply]
Jika khawatir terkena popup lainnya yang mungkin penting, ini untuk yang bner2 spesifik banget

```adblock
eworldtrade.com##.exit-intent-popup-ssn:has(> .newsletter-ssn):has-text(/signup today/i)
```
